### PR TITLE
Add gRPC Dockerfile

### PR DESCRIPTION
**Pull request by cmuench**
Adds a small Dockerfile to enable gRPC in ddev projects.

### CODE REVIEW

**Review by rfay — CHANGES_REQUESTED** [review]
Thanks for this! 
* Please add a little detail to the README saying why you'd want to use this.
* Please add an entry in the top-level README.md
Thanks!

**Comment by cmuench** [comment]
@rfay ey ey, Sir. :-)

**Comment by rfay** [comment]
Thanks so much! I'm on vacation but look forward to trying this out when I return after Christmas or in the new year.

**Comment by cmuench** [comment]
@rfay ok. Have nice holidays.

**Review by rfay — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ General information on how to do additional services and some additional example
 
 * [Laravel Queue-Worker](web-container-dockerfiles/laravel-queue-worker) (This is also a good example of adding an additional process to supervisord,)
 * [Stripe CLI](web-container-dockerfiles/stripe-cli) (This is also a good example of adding any non-standard Debian repository.)
+* [gRPC](web-web-container-dockerfiles/grpc)
```
Comment (rfay) [inline README.md:50]:
```suggestion
* [gRPC](web-web-container-dockerfiles/grpc) (This is also a good example of adding a pecl module that is not supported via apt-get.)
```

**Pull request merged**